includes/diagnostics.php

```diff
@@ -9,6 +9,8 @@
 
 global $wp_object_cache;
 
+/** @var \Rhubarb\RedisCache\Plugin $roc */
+
```

Comment:
Let's drop the extra line.

Reply:
This "defines" $roc

I leave it 100% up to you.